src/domain/useCase/createCustomService.go

```diff
@@ -63,7 +63,12 @@ func CreateCustomService(
 	vhostQueryRepo repository.VirtualHostQueryRepo,
 	createDto dto.CreateCustomService,
 ) error {
-	_, err := servicesQueryRepo.ReadByName(createDto.Name)
+	shouldIncludeMetrics := false
+	readInstalledDto := dto.ReadInstalledServicesItemsRequest{
+		ServiceName:          &createDto.Name,
+		ShouldIncludeMetrics: &shouldIncludeMetrics,
+	}
+	_, err := servicesQueryRepo.ReadOneInstalledItem(readInstalledDto)
```

Comment:
ReadOne doesn't tell me that it must returns at least one. It just says to me it'll return one. However, it may return nothing and the err still be nil. I would replace it with MustReadOne to avoid any possible confusion. If you use MustRead, you may also ditch the one and just use the first item it returns, which is even easier and more reusable but not obligatory.

Reply:
We discussed this in the channel, and it was decided that the functions will be named `ReadFirst` instead of `ReadOne` to meet this need without causing confusion.